packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -2517,14 +2517,20 @@ const EditorPanel = () => {
                               )}
                             </select>
                           </label>
-
                           <CheckBox
                             value={config.xAxis.manual}
                             section='xAxis'
                             fieldName='manual'
                             label='Manual Ticks'
                             updateField={updateField}
                           />
+                          <CheckBox
```

Comment:
Only show this setting when the scale type is a date scale.

Reply:
Fixed!